tests/nightly/nightly-autorun.sh

```diff
@@ -16,8 +16,10 @@
 
 # Get the latest code from vmware/vic repo
 cd ~/go/src/github.com/vmware/vic
-git fetch
+git clean -fd
```

Comment:
Yes @anchal-agrawal, this will remove that `nightly_test_secrets.yml` file.  @mhagen-vmware, is there a way we can move that nightly secrets file somewhere else and update the drone execution command to use the new location?
I want to clone in the vic-internal repo somewhere on the nightly executor machine and update this but I'm still having some issues getting repo 'read' privileges for a user that is not me for this machine.